src/styles/main.scss

```diff
@@ -1,3 +1,66 @@
-body {
+$gap-size: 1px;
+$ceil-width: 100px;
+$ceit-height: 100px;
+$days: (
+  "mon": 0,
+  "tue": 1,
+  "wed": 2,
+  "thu": 3,
+  "fri": 4,
+  "sat": 5,
+  "sun": 6,
+);
+
+$month-length: 28, 29, 30, 31;
+
+html {
+  font-size: 30px;
   margin: 0;
+  padding: 0;
+}
+
+.calendar {
+  margin: 0 auto;
+  box-sizing: border-box;
+  display: flex;
+  gap: $gap-size;
+  flex-wrap: wrap;
+  width: $ceil-width * 7 + $gap-size * 6 + 20px;
+  padding: 20px 10px 10px;
+
+  &__cell {
+    box-sizing: border-box;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: #eee;
+    border: 1px solid black;
+    width: $ceil-width;
+    height: $ceit-height;
+    font-size: 30px;
+    transition: all 0.5s;
```

Comment:
It's better to specify what properties should apply the transition.
![image](https://github.com/mate-academy/layout_calendar/assets/63243136/4494204d-747e-41f9-aa52-5cc82400a5c8)